packages/core/diagnostic/src/diagnostic.js

```diff
@@ -119,14 +119,18 @@ export function anyToDiagnostic(input: Diagnostifiable): Array<Diagnostic> {
 /** Normalize the given error into a diagnostic. */
 export function errorToDiagnostic(
   error: ThrowableDiagnostic | PrintableError | string,
-  realOrigin?: string,
+  defaultValues: {
+    origin?: ?string,
+    filePath?: ?string,
+    ...
```

Comment:
Could this be an exact object type instead?

Reply:
Sure